tests/contrib/mcp/test_mcp_llmobs.py

```diff
@@ -146,6 +161,55 @@ def test_llmobs_client_server_tool_error(mcp_setup, mock_tracer, llmobs_events,
     )
 
 
+def test_server_initialization_span_created(mcp_setup, mock_tracer, llmobs_events):
+    """Test that server initialization creates a span and LLMObs event with custom client info."""
+    from mcp.server.fastmcp import FastMCP
+    from mcp.shared.memory import create_connected_server_and_client_session
+    from mcp.types import Implementation
+
+    mcp_server = FastMCP("TestInitServer")
+
+    async def run_test():
+        client_info = Implementation(name="test-client", version="1.2.3")
+        async with create_connected_server_and_client_session(mcp_server._mcp_server, client_info=client_info):
+            pass
```

Comment:
i think this could be best as a pytest fixture to declutter this test function a bit